build.sbt

```diff
@@ -151,8 +152,8 @@ lazy val `nsdb-cluster` = project
       val confDir = baseDirectory.value / "src/main/resources"
 
       for {
-        (file, relativePath) <- (confDir.*** --- confDir) x relativeTo(confDir)
-      } yield file -> s"/opt/${(packageName in Docker).value}/conf/$relativePath"
+        (file, relativePath) <- (confDir.*** --- confDir) pair (relativeTo(confDir), false)
+      } yield file -> s"/opt/${name.value}/conf/$relativePath"
```

Comment:
This change override a the `(packageName in Docker).value` update.

Also line [162](https://github.com/radicalbit/NSDb/pull/18/files#diff-fdc3abdfd754eeb24090dbd90aeec2ceR162) should be updated with `pair (relativeTo(confDir), false)`